server/sonar-webserver-webapi/src/main/java/org/sonar/server/newcodeperiod/ws/SetAction.java

```diff
@@ -151,12 +151,7 @@ public void handle(Request request, Response response) throws Exception {
         if (branchKey != null) {
           branch = getBranch(dbSession, project, branchKey);
           dto.setBranchUuid(branch.getUuid());
-        } else if (isCommunityEdition) {
```

Comment:
Why do we use Check for isCommunityEdition, and how do we identify the product edition